src/components/JourneyTimeline/JourneyTimeline.tsx

```diff
@@ -182,10 +182,13 @@ const TransferSegment = (props: {
 };
 
 const JourneyTimeline = (props: Props) => {
-    const { journey } = props;
+    const { segments } = props;
+    if (segments.length === 0) {
+        return <>No route found for this mode</>;
```

Comment:
I don't know if "mode" is quite the right word here. Maybe we could just return `null` here since the comparison rows above make it pretty clear that there's nothing here.

I have an idea for a placeholder graphic that we could use here — we could create a fake `JourneyTimeline` strip object in a light grey that kind of fades into the mist towards the bottom of the page. Probably something to explore after this PR though!